lms/djangoapps/instructor/tests/test_proctoring.py

```diff
@@ -36,24 +34,74 @@ def setUp(self):
         self.instructor = AdminFactory.create()
         self.client.login(username=self.instructor.username, password="test")
 
-        CourseFinanceAdminRole(self.course.id).add_users(self.instructor)
+    def setUpCourseUrl(self, course):
```

Comment:
Is there any specific reason you used camel case here?

Reply:
No, there is no specific reason.

Reply:
So, I think we should go with convention of snake_case. (sounds a bit scary though)